easybuild/easyconfigs/o/OpenMPI/OpenMPI-5.0.0rc10-GCC-12.2.0.eb

```diff
@@ -0,0 +1,35 @@
+name = 'OpenMPI'
+version = '5.0.0rc10'
```

Comment:
final release of `5.0.0` is out now, we should probably also move this to GCC 13.2 while we're at it...

Reply:
I made a PR to update PMIx and OpenMPI to `5.0.1` and use GCC 13.2.0 in https://github.com/boegel/easybuild-easyconfigs/pull/92